docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -1394,6 +1394,50 @@
         }
       ]
     },
+    {
+      "metadata": {
+        "id": "9zKWsFm7vjFw",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Notice above that we didn't wrap the closure in parentheses when passing it into the `sorted` function. This is because if the last (or only) argument of a function is a closure, the parentheses for it are optional. For a more clear example:"
```

Comment:
This change is the addition of showing how you don't need to put the closure inside the parameter of the function.